### ISSUE TITLE
Error while testing ICG

### ISSUE THREAD

**Issue by danielfaguilar** [sheetalsh456/Compiler-Design#1]
Hi, I'm new in Compiler Design and I was testing the ICG and after executed the following commands without errors:
lex      s_parser.l
yacc   s_parser.y
./a.out test1.c

I got the following error in the terminal output:
a = 5
b = 6
[1]    3641 abort      ./a.out test1.c

Seems like the problem comes from the assignation.

**Comment by SenbagapriyaS** [comment]
Yes..I too encountered this error when I try to compile in MacOsX..But it's working fine in Ubuntu

**Comment by ashishch164** [comment]
When i tried to execute third command of ICG, then i got this error "gcc: error: y.tab.c: No such file or directory". So, can you help me out with it and please upload y.tab.c file in ICG folder.

**Comment by SenbagapriyaS** [comment]
Can u plz send the screenshot of the error in the terminal?

**Comment by ashishch164** [comment]
i was able to resolve the issue. I searched on google and I found the solution, so now its working fine.

**Comment by SenbagapriyaS** [comment]
Cool!

**Comment by alokumarjaiswal** [comment]
I have made some changes. Please have a look, everything is working fine as of now.
